clients/admin-ui/src/types/api/models/UserPermissionsPlusResponse.ts

```diff
@@ -7,8 +7,10 @@ import type { ScopeRegistryEnum } from "./ScopeRegistryEnum";
 
 /**
  * Response after creating, editing, or retrieving a FidesUserPermissions record.
+ *
+ * Overrides in Plus to use the Plus ScopeRegistryEnum so Plus scopes are returned.
  */
-export type UserPermissionsResponse = {
+export type UserPermissionsPlusResponse = {
```

Comment:
I think this should be safe, but here is what is going on:

* the backend is overwriting the permission response to a plus specific response with plus specific scopes.
* the actual type of the result is the same (still an array of `roles` and `total_scopes`), which is why the only change in this file is the name of the type. The contents of those arrays are different between plus and OSS though (where plus is a superset of OSS)
* we always run our types off of plus since it has all of the types, and this has worked well for us in the past. I think this should still be okay since there isn't really a need for the FE to know which scopes belong to plus vs OSS, at least not right now

Reply:
This seems safe to me for now. If we run into any issues we can update plus to return a type that's only the OSS scopes